e2e/e2e_test.go

```
@@ -69,54 +65,71 @@ var _ = Describe("Central", func() {
 		Expect(err).ToNot(HaveOccurred())
 		adminClient, err := fleetmanager.NewClient(fleetManagerEndpoint, adminAuth)
 		adminAPI = adminClient.AdminAPI()
-
 		Expect(err).ToNot(HaveOccurred())
 
+		GinkgoWriter.Printf("Current time: %s\n", time.Now().String())
+		printNotes(notes)
 	})
 
 	Describe("should be created and deployed to k8s", func() {
 		var err error
-
-		centralName := newCentralName()
-		request := public.CentralRequestPayload{
-			CloudProvider: dpCloudProvider,
-			MultiAz:       true,
-			Name:          centralName,
-			Region:        dpRegion,
-		}
-
 		var createdCentral *public.CentralRequest
 		var namespaceName string
+
 		It("created a central", func() {
+			centralName := newCentralName()
+			request := public.CentralRequestPayload{
+				CloudProvider: dpCloudProvider,
+				MultiAz:       true,
+				Name:          centralName,
+				Region:        dpRegion,
+			}
 			resp, _, err := client.PublicAPI().CreateCentral(context.Background(), true, request)
-			createdCentral = &resp
 			Expect(err).To(BeNil())
+			createdCentral = &resp
+			notes = []string{
+				fmt.Sprintf("Central name: %s", createdCentral.Name),
+				fmt.Sprintf("Central ID: %s", createdCentral.Id),
+			}
+			printNotes(notes)
 			namespaceName, err = services.FormatNamespace(createdCentral.Id)
 			Expect(err).To(BeNil())
 			Expect(constants.CentralRequestStatusAccepted.String()).To(Equal(createdCentral.Status))
 		})
 
 		It("should transition central's state to provisioning", func() {
+			if createdCentral == nil {
```

Fail fast, when the central could not be created.